src/components/Sortable/Sortable.types.ts

```diff
@@ -1,22 +1,22 @@
 export interface SortableProps {
-  axis: 'x' | 'y' | 'xy'
-  className: string
-  distance: number
-  lockAxis: string
-  helperClass: string
-  hideDragHandles: boolean
-  transitionDuration: number
-  contentWindow: any
-  onSortStart: () => void
-  onSortMove: () => void
-  onSortEnd: (...args: any[]) => void
-  shouldCancelStart: () => void
-  pressDelay: number
-  useDragHandle: boolean
-  useWindowAsScrollContainer: boolean
-  hideSortableGhost: boolean
-  lockToContainerEdges: boolean
-  lockOffset: number | string | Array<string | number>
-  getContainer: () => void
-  getHelperDimensions: () => void
+  axis?: 'x' | 'y' | 'xy'
+  className?: string
+  distance?: number
+  lockAxis?: string
+  helperClass?: string
+  hideDragHandles?: boolean
+  transitionDuration?: number
+  contentWindow?: any
+  onSortStart?: () => void
+  onSortMove?: () => void
+  onSortEnd?: (...args: any[]) => void
+  shouldCancelStart?: () => void
+  pressDelay?: number
+  useDragHandle?: boolean
+  useWindowAsScrollContainer?: boolean
+  hideSortableGhost?: boolean
+  lockToContainerEdges?: boolean
+  lockOffset?: number | string | Array<string | number>
+  getContainer?: () => void
+  getHelperDimensions?: () => void
```

Comment:
None of these are required and create problems when trying to use the component internally the same way we use it externally. It is perfectly valid to make something sortable by wrapping it with `<Sortable></Sortable>` with no props.